src/sentry/relay/config/ai_model_costs.py

```diff
@@ -0,0 +1,153 @@
+from typing import TypedDict
+
+
+class AIModelCost(TypedDict):
+    modelId: str
+    forCompletion: bool
+    costPer1kTokens: float
+
+
+class AIModelCosts(TypedDict):
+    version: int
+    costs: list[AIModelCost]
+
+
+def ai_model_costs_config() -> AIModelCosts:
+    return {
+        "version": 1,
+        "costs": list(
+            map(
+                lambda row: {
+                    "modelId": row[0],
+                    "forCompletion": row[1],
+                    "costPer1kTokens": row[2],
+                },
```

Comment:
```suggestion
        "costs": nit

```suggestion
        "costs": [
            {
                "modelId": row[0],
                "forCompletion": row[1],
                "costPer1kTokens": row[2],
            }
            for row in [
```
```